notebooks/analyze_model_output.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    model_weights_path = '/gpfs/data1/cmongp1/mpaliyam/street2sat/yolov5/runs/train/exp18/weights/best.pt'  
</pre><p>Not relevant for this PR, but for future it would be good to track the weights in dvc also!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>